src/qt/optionsmodel.cpp

```diff
@@ -54,7 +55,7 @@ void OptionsModel::Init(bool resetSettings)
 
     // Window
     if (!settings.contains("fHideTrayIcon"))
-        settings.setValue("fHideTrayIcon", false);
+        settings.setValue("fHideTrayIcon", !QSystemTrayIcon::isSystemTrayAvailable());
```

Comment:
I'd leave this alone. Or at the very least, don't set any value if there's no systray.

Reply:
If it'll be leaved alone the first call of the `OptionsModel` constructor sets `fHideTrayIcon` to the `false` anyway.
Why cannot it be initialized to the `true` on the GNOME environment?

Reply:
I'd say that's another PR, like "gui: Enable system tray icon by default if available"

Reply:
@hebasto Because the environment is not necessarily a permanent thing. Someone might run it on GNOME, and then run it on KDE.

Systems without a systray should just behave as if the option doesn't exist at all IMO.